templates/gitlab-multi-runner/0/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '2'
+
+services:
+
+  gitlab-runner-config:
+    image: gitlab/gitlab-runner:alpine
```

Comment:
Please, use concrete version instead implicit latest....Latest is no reproducible, please use gitlab/gitlab-runner:alpine-v9.3.0 or the needed version...